src/scripts/findOffers.ts

```diff
@@ -1,9 +1,179 @@
-const findOffers = () => {
-    console.log('Scrapping...')
+import puppeteer, { Browser, Page, ElementHandle } from 'puppeteer';
+import { Cluster } from 'puppeteer-cluster';
+import ScraperOptions from '../interfaces/ScraperOptions';
+import JobOffer from '../interfaces/JobOffer';
+import writeToJson from '../helpers/writeToJson';
 
-    setTimeout(() => {
-        console.log('10 offers found.')
-    }, 3000)
+const pptOptions = {
+  headless: false,
+  args: [`--window-size=1920,1080`, '--no-sandbox'],
+};
+
+const selector = 'div.css-ic7v2w';
+
+async function delay(miliseconds: number): Promise<void> {
+  return new Promise(function (resolve) {
+    setTimeout(resolve, miliseconds);
+  });
+}
+
+async function navigateTo(url: string, page: Page): Promise<void> {
+  console.log(`Navigating to: ${url} ...`);
+
+  await page.goto(url, { waitUntil: 'networkidle2' });
+  await page.setViewport({ width: 1920, height: 1080 });
 }
 
-findOffers()
\ No newline at end of file
+async function extractItems(page: Page): Promise<string[]> {
```

Comment:
**nitpick:** Such function like `extractItems` could be also a part of your scrapper. Currently you hardcoded specific selector value. When you are gonna scrap second job board in another task, you would have to duplicate this function. To avoid this you should have something like: 
```javascript
extractItems(params: {
page: Page,
targetSelector: string,
searchedElementSelector: string,
searchedElementAttr: string
}) => {
const items: string[] = await params.page.$$eval(
    params.targetSelector,
    (elements: HTMLElement[]) =>
      elements.map((element) => element.querySelector(params.searchedElementSelector)[params.searchedElementAttr])
  );

  return items;
}
```